ReSharper.FSharp/src/FSharp.Psi/src/Impl/DeclaredElement/CompilerGenerated/FSharpGeneratedPropertyAccessor.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.Cache2.Parts;
+using JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.Pointers;
+using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.Pointers;
+
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.DeclaredElement.CompilerGenerated
+{
+  internal class FSharpGeneratedPropertyAccessor : FSharpGeneratedMethodBase, IFSharpGeneratedFromProperty
+  {
+    private readonly bool myIsGetter;
+    private readonly IAccessor myAccessor;
```

Comment:
Why does an accessor instance hold a reference to another accessor?

Reply:
Just for proof of concept. I'm going to make the appropriate changes in the platform.